### PR TITLE
Remove support for classRef/@expand and classSpec/@generate

### PR DESCRIPTION
**Pull request by dmj**
See discussion in https://github.com/TEIC/TEI/issues/2369 -- it makes things easier for us to remove support for classRef/@‍expand and classSpec/@‍generate and be upfront about it.

### CODE REVIEW

**Comment by martindholmes** [comment]
@sydb would like to wait to see how the TEI ticket https://github.com/TEIC/TEI/issues/2369 ends up before doing this, so we're holding off for now.

File: Schemas/ploddSchemaSpecification.odd [hunk 1]
```diff
@@ -392,6 +393,7 @@
             <attDef ident="rend"      mode="delete"/>
             <attDef ident="style"     mode="delete"/>
             <attDef ident="rendition" mode="delete"/>
+            <attDef ident="generate"  mode="delete"/>
```
Comment (sydb) [inline Schemas/ploddSchemaSpecification.odd:396]: TEI has committed to ditching this attr. The intent was just to make it a no-op, i.e. to ignore it. But I see no strong reason not to disallow it, as done here.

File: Schemas/ploddSchemaSpecification.odd [hunk 2]
```diff
@@ -375,6 +375,7 @@
             <attDef ident="rend"      mode="delete"/>
             <attDef ident="style"     mode="delete"/>
             <attDef ident="rendition" mode="delete"/>
+            <attDef ident="expand"    mode="delete"/>
```
Comment (sydb) [inline Schemas/ploddSchemaSpecification.odd:378]: TEI has not committed to dropping this attr on https://github.com/TEIC/TEI/issues/2369. That said, a) they probably will; and b) until they either drop it or clearly define its semantics, there is no need for us to fret over how to process it. So, at least for now (and probably permanently) it makes sense to me to disallow it.

**Review by sydb — APPROVED** [review]

**Review by martindholmes — COMMENTED** [review]
@HelenaSabel and I merged dev and tested, and all looks good. We'll merge now.

**Pull request merged**